Homework_6.py

```diff
@@ -0,0 +1,14 @@
+from RUN import Run_1
+from RUN import Run_2
+from RUN import Run_3
+
+# Оформите решение задач из прошлых домашних работ в функции.
+# Напишите функцию runner.
+# runner() – все фукнции вызываются по очереди
+# runner(‘gen_numbers’) – вызывается только функцию gen_numbers.
+# runner(‘func’, ‘func1’...) - вызывает все переданные функции
+Run_1.runner_1()
```

Comment:
it's not exactly what is required. we should have one function that can work in these 3 modes. 
you can use getattr to avoid all if's in your runners